### PR TITLE
feat: polish README.md, resolves #1

### LINKED ISSUE #1: Update README.md
It's fork from upstream and outdated.

### PR DESCRIPTION
**Pull request by delbertbeta**
resolves #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,133 +1,90 @@
...
+<br />
 
-### Making APIs Ready
+### Deploy
 
-4. Get Discord API Key via : https://discord.com/developers/applications
+#### Get the codes
 
-After you create a API key then you will get to see a token once so you should copy that and keep it safely somewhere, better to use Google Keep - https://keep.google.com/ for this.
+This project is writeen in TypeScript, before deploy, ensure you have NodeJS (>= v20 is recomment) installed. If you don't have NodeJS, [fnm](https://github.com/Schniz/fnm) is the NodeJS installer recommended.
```
Comment (RuMaxwell) [inline README.md]: typo: is **written**; is **recommended**

File: README.md [hunk 2]
```diff
@@ -1,133 +1,90 @@
...
-### Setup Initial Coding Workspace
-Execute this commands on your IDE on a directory you want to work on
-1. ```npm init```
-2. ```npm install discord.js```
-3. ```npm install @google/generative-ai```
+- [x] 😀 Chat using texts
+- [x] 👋🏼 Chat with context
+- [x] 🥷🏼 Clear context with `/clear` command
+- [x] 🚀 Streaming output
+- [ ] Multimode support
```
Comment (RuMaxwell) [inline README.md]: Multimodal

File: README.md (hunk 1 above)
Comment (delbertbeta) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (delbertbeta) [reply README.md]: done

File: README.md [hunk 3]
```diff
@@ -1,133 +1,90 @@
...
-3. After this you will get a Generated URL for bot invite.![alt text](Images/image-2.png)
+<br />
 
-Congrats on making your own Gemini Bot.
+#### Add Gemini bot to your Discord Server
 
-Thank you for Reading till the end, I wish to see your own bot live. 
+1. Open https://discord.com/developers/applications and select the bot you just created.
+2. Click `OAuth2` in the nav.
+3. In `OAuth2 URL Generator`, check the folowing options:
```
Comment (RuMaxwell) [inline README.md]: following

Comment (RuMaxwell) [reply README.md]: Line 71 还有一个

File: README.md [hunk 4]
```diff
@@ -1,133 +1,90 @@
...
+   2. `Create Public Threads`
+   3. `Manage Messages`
+   4. `Embed Links`
+   5. `Attach Files`
+   6. `Read Message History`
+   7. `Mention Everyone`
+   8. `Use Slash Commands`
+   9. `View Channels`
+5. `Integration Type` choose `Guild Install`
+6. Copy the `Generated URL` and open it in your browser, now you can add it into your discrod server.
```
Comment (RuMaxwell) [inline README.md]: discord

Comment (delbertbeta) [reply README.md]: 妈的，得装个 spell check 插件才行了

**Review by RuMaxwell — APPROVED** [review]

**Pull request merged**